easybuild/easyblocks/m/molpro.py

```diff
@@ -106,6 +106,13 @@ def configure_step(self):
                     self.cfg.update('configopts', "-mpp -mppbase %s" % os.environ['MPI_INC_DIR'])
                 else:
                     raise EasyBuildError("$MPI_INC_DIR not defined")
+                if self.toolchain.mpi_family() == toolchain.OPENMPI:
+                    self.cfg.update('configopts', "-openmpi")
+                elif self.toolchain.mpi_family() == toolchain.INTELMPI:
+                    self.cfg.update('configopts', "-intel-mpi")
+                else:
+                    raise EasyBuildError("Failed to find out which MPI toolchain is in use: %s"
+                                         % self.toolchain.mpi_family())
```

Comment:
@akesandgren We have several `Molpro` easyconfig that use `dummy` as toolchain, this change will break the installation of each of those...

Reply:
Why? It already below a
if self.toolchain.options.get('usempi', None):

Reply:
And moreover, all those taht use dummy toolchain also use precompiled_binaries = True and won't be in that part of the code anyway.

Reply:
OK, sorry, I overlooked that part...